orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -49,10 +49,12 @@
 	private static String releaseVersion;
 	private static GKInstance instanceEditInst;
 	private static GKInstance speciesInst;
-	private static Map<GKInstance,GKInstance> manualEventToNonHumanSource = new HashMap<>();
-	private static List<GKInstance> manualHumanEvents = new ArrayList<>();
+	private static Map<GKInstance, GKInstance> eventsAlreadyInferredMap = new HashMap<>();
+	private static List<GKInstance> eventsAlreadyInferred = new ArrayList<>();
 	private static StableIdentifierGenerator stableIdentifierGenerator;
 	private static OrthologousPathwayDiagramGenerator orthologousPathwayDiagramGenerator;
+	private static final String summationText = "This event has been computationally inferred from an event that has been demonstrated in another species.<p>The inference is based on the homology mapping from PANTHER. Briefly, reactions for which all involved PhysicalEntities (in input, output and catalyst) have a mapped orthologue/paralogue (for complexes at least 75% of components must have a mapping) are inferred to the other species. High level events are also inferred for these events to allow for easier navigation.<p><a href='/electronic_inference_compara.html' target = 'NEW'>More details and caveats of the event inference in Reactome.</a> For details on PANTHER see also: <a href='http://www.pantherdb.org/about.jsp' target='NEW'>http://www.pantherdb.org/about.jsp</a>";
```

Comment:
I'm not sure the "target = 'NEW'" attribute that is used two times is allowed in modern HTML.  For a new window or tab, 'target=_blank' is the attribute value to use:

https://www.w3schools.com/tags/att_a_target.asp

Reply:
Could this string be split on to several lines? This is a bit wide, even for me.